api/src/movie/movie.controller.ts

```diff
@@ -27,6 +27,16 @@ export class MovieController {
     return this.movieService.getMovies(req.user.id, movieId);
   }
 
+  @Get('/:userId/:listId')
+  async getWatchedListMovies(
```

Comment:
`userId` and `listId` should probably be query params.
Also, can we call this `getMovies`?

Reply:
Oh, I just saw that we already have a `getMovies`. We should combine that with this one.